### ISSUE TITLE
Checking URL results in white page

### ISSUE THREAD

**Issue by Cerberooo** [w3c/i18n-checker#82]
If I test the url `https://www.tu-chemnitz.de/wirtschaft/bwl4` with https://validator.w3.org/i18n-checker only a white page is shown. I think this could be that the page has no valid HTML due to a Byte-Order-Mark problem.

**Comment by lcarcone** [comment]
@Cerberooo this problem is due to some <button> elements nested inside a <a>. If I remove them, the validator returns results.
Here is a page that can help you resolve this issue in your document:
https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5

Regards

**Comment by lcarcone** [comment]
I meant 'some button elements nested inside an element a'

**Comment by Cerberooo** [comment]
@lcarcone So it's intended, that an invalid document results in a white page?

**Comment by r12a** [comment]
I agree that that's not ideal.  @lcarcone I think we have some messages that reported an inability to parse the page (at least for certain things), but i'm not aware of the details, or how this fits with that.

**Comment by lcarcone** [comment]
@Cerberooo I'm not a developer of this tool so I cannot say if it is intended or not, I let someone else answer this question.
Regards

**Comment by r12a** [comment]
@Cerberooo i'll get someone to post an update here if and when there are further developments.

**Comment by xfq** [comment]
This scenario falls under the "reconstruct active formatting elements" step within the HTML5 parsing algorithm. The original logic, when searching for an active formatting element, could potentially halt at a marker. In the code this marker is represented by the integer 300, rather than a standard DOM node, so `cloneNode` won't work.

Should be relatively trivial to fix.
